rfcs/text/0000-bundle-types.md

```diff
@@ -0,0 +1,87 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC: Bundle Types
+
+## Summary
+
+Formalizes data bundle types, definitions, target users/consumers, and specific use cases to improve clarity and
+confidence among DCP developers when working with the DCP data model.
+
+## Author(s)
+
+* [Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)
+* [Brian Hannafious](mailto:bhannafi@ucsc.edu)
+* [Hannes Schmidt](mailto:hannes@ucsc.edu)
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Shepherd
+
+* [Andrey Kislyuk](mailto:akislyuk@chanzuckerberg.com)
+
+## Motivation
+
+There are currently no documented definitions for what a "bundle" is in the HCA DCP, including no definitions of what
+they are, what they contain, or who they support. Lack of clarity causes confusion for DCP developers and data consumers
+about how the data and metadata in the DCP are structured and organized.
+
+### User Stories
+
+1. As a tool developer, I would like to identify and get raw data in the DSS so that I can run my data processing and
```

Comment:
Just a syntax note, numbering is not incrementing.

Reply:
@TimothyTickle that is intentional. Markdown supports automatic numbered list incrementing, so that items can be re-arranged more easily. The rendered version increments the list properly.